### DISCUSSION TITLE
Diataxis - a strategy for creating and organising effective documentation

### DISCUSSION THREAD

**Discussion by ljwolf** [pysal/pysal#1289] [Ideas]
Hey folks, ran into this today on HN: 

https://diataxis.fr

It looks like a neat way to understand how to create and curate technical documentation, based on two axes of (pedagogical <-> practical) and (theoretical <-> implementational). So, for example, a how-to-guide should be practical and focus on exact implementations (avoiding the theory behind techniques as far as possible), while a tutorial would include the theoretical content about estimators and algorithms. 

Thought it might be useful to record. Before, I'd just dump this into our gitter 🤠

**Comment by jlaura** [comment]
@ljwolf I hope all is well! 

You might also like [https://documentation.divio.com](https://documentation.divio.com). Looks like Diataxis was derived from Divio (which made the hacker news rounds a while ago I think?)

**Comment by ljwolf** [comment]
Thanks! Hope all is well with you, too 😄 

It does indeed look like Diataxis is a Divio derivative... and Divio looks way more fleshed out!
